pkg/gui/controllers/branches_controller.go

```diff
@@ -412,6 +410,66 @@ func (self *BranchesController) copyPullRequestURL() error {
 	return nil
 }
 
+func (self *BranchesController) copyBranchAttributeToClipboard(branch *models.Branch) error {
+	return self.c.Menu(types.CreateMenuOptions{
+		Title: self.c.Tr.Actions.CopyBranchAttributeToClipboard,
+		Items: []*types.MenuItem{
```

Comment:
It's better to disable the options that don't make sense than allow the user to pick the and get an error.

E.g. if the branch has no upstream we can disable the copying of both branch and PR URLs.

Take a look at [disabling other menu items](https://github.com/skolj/lazygit/blob/6821eedf69b7c237ce0fa011d667a3f65a0f66a9/pkg/gui/controllers/files_controller.go#L836).

Reply:
I didn't realize that existed. Thanks for the pointer!